autoload/clap/provider/git_diff_files.vim

```diff
@@ -0,0 +1,23 @@
+" Author: KITAGAWA Yasutaka <kit494way@gmail.com>
+" Description: List the files which is managed by git and have uncommitted changes.
```

Comment:
From the help of `git status`, `-u[<mode>], --untracked-files[=<mode>]`, `git_untracked_files` is a better name than `git_diff_files`? What do you think?

Reply:
Option `-uno` means "Show no untracked files", so `git_diff_files` is better.
Behavior of the option `-u[<mode>]` depends on `mode`.
Default mode `all` shows untracked files, but mode `no` does not show untracked files.

Reply:
Ok, I have no problem then. Can you please also update the README and help doc?

Reply:
Yes, I will.

Reply:
I updated the README and help doc.